inginious/frontend/plugins/user_management/static/json/collections_info.json

```diff
@@ -0,0 +1,85 @@
+{
+  "aggregations": [
+    {
+      "path": "students",
+      "index_array": [0]
+    },
+    {
+      "path": "tutors",
+      "index_array": [0]
+    },
+    {
+      "path": "groups.students",
+      "index_array": [0,1]
+    }
+  ],
+  "analytics": [
+    {
+      "path": "username",
+      "index_array": []
+    }
+  ],
+  "custom_tests": [
+    {
+      "path": "none"
```

Comment:
Setting the `path` to `none` is quite confusing, I understand this file is to tell the manager where to look for the username. Two options, these collections are removed from this file and only the ones that do not follow a pattern are kept, or you follow set the actual `path` for each collection. Choose the best one.

Reply:
It is better kept the path for each collection because there are collections that use username, but they have a special purpose like sessions, users_changes or custom_tests. Nnormally they just store temporal data and it is better to indicate do not touch them